src/lib/automap.ahk

```diff
@@ -0,0 +1,47 @@
+﻿class automap {
+
+}
```

Comment:
Is this a class?

Reply:
This is the beginning of a class. it's technically useless at this point but I wanted to leave it in. I'm working on consolidating the script functions into classes, nothing to see here... yet.